sympy/printing/repr.py

```diff
@@ -244,7 +244,9 @@ def _print_Predicate(self, expr):
         return "%s(%s)" % (expr.__class__.__name__, self._print(expr.name))
 
     def _print_AppliedPredicate(self, expr):
-        return "%s(%s, %s)" % (expr.__class__.__name__, expr.func, expr.arg)
+        return "%s(%s, %s)" % (
+            expr.__class__.__name__, expr.func, self.reprify(expr.args, ", ")
```

Comment:
I think this should be
```suggestion
            expr.__class__.__name__, self._print(expr.func), self.reprify(expr.args, ", ")
```
to avoid going through `str` &rarr; `sstr` &rarr; `StrPrinter.doprint` &rarr; `StrPrinter._print`.


Can `expr.args` be empty? If so, this should be
```suggestion
            expr.__class__.__name__, self.reprify((expr.func,) + expr.args, ", ")
```
so as not to emit a trailing comma when it is.